core/server/common/src/main/java/alluxio/cli/validation/SecureHdfsValidationTask.java

```diff
@@ -41,23 +48,31 @@ public SecureHdfsValidationTask(String process) {
   }
 
   @Override
-  public boolean validate() {
+  public boolean validate(String[] args, Map<String, String> optionsMap) {
+    if (!validatePrincipalLogin()) {
+      System.err.format("Principal login test failed.");
+      return false;
+    }
+    if (!validateHdfsSettingParity(optionsMap)) {
```

Comment:
Can you put this into a different task? hdfs configuration consistency and security settings feels like two different things to check.

Reply:
They are different, but they are both required for Alluxio to connect to secure HDFS. Therefore, I feel it's more convenient to put them together. Or we can great another target that runs both tasks?

Reply:
You can always use prefix to run a group of validation tasks. e.g. If you do `alluxio validateEnv local master.hdfs` it will run both `master.hdfs.security` and `master.hdfs.consistency`.